### PR TITLE
Check for existence before calling Google Analytics in index template

### PR DESCRIPTION
**Pull request by lmorchard**
Submitting this, since I think I prematurely merged PR #577 and this looked like one of the last issues

### CODE REVIEW

File: testpilot/frontend/templates/testpilot/frontend/index.html [hunk 1]
```diff
@@ -24,7 +24,9 @@
     m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
     })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
 
-    ga('create', 'UA-49796218-34', 'auto');
+    if (typeof(ga) !== 'undefined') {
+        ga('create', 'UA-49796218-34', 'auto');
+    }
```
Comment (meandavejustice) [inline testpilot/frontend/templates/testpilot/frontend/index.html:34]: Would maybe be nice to those an `else console.warn('You have google analytics blocked. We understand. Take a look at our privacy policy to see how we handle your data.')`

**Comment by meandavejustice** [comment]
Can we also add this check here https://github.com/mozilla/testpilot/blob/master/testpilot/frontend/static-src/app/main.js#L71 ?

**Comment by lmorchard** [comment]
That check is already [here](https://github.com/mozilla/testpilot/blob/master/testpilot/frontend/static-src/app/main.js#L66), though. (line 66, before that ga() call on 71)

**Comment by meandavejustice** [comment]
@lmorchard Wow, I totally over looked that, sorry!

**Pull request merged**